lib/streamlit/testing/v1/element_tree.py

```diff
@@ -78,6 +77,8 @@
 from streamlit.runtime.state.safe_session_state import SafeSessionState
 
 if TYPE_CHECKING:
+    from pandas import DataFrame as PandasDataframe
```

Comment:
Renamed this, otherwise it is a bit confusing since there is also another DataFrame class in this Fiel.